src/components/SearchElement/SearchAirlineElement/SearchAirlineElement.js

```diff
@@ -57,31 +61,39 @@ const  SearchAirlineElement = (props) => {
                             elementType='input'
                             elementConfig={iataInputConfig}
                         />
-                    </div>
-                </div>    
-                <div className="col-md">                
-                    <div className={classes.card}> 
                         <Input 
                             value={icao}
                             changed={(e)=>setICAO(e.target.value)}          
                             elementType='input' 
                             elementConfig= {icaoInputConfig}                     
-                        />                         
+                        />
+                    </div>
+                </div>    
+                <div className="col-md">                
+                    <div className={classes.card}>                                                  
+                        <Input
+                            value={fleet}
+                            changed={(e)=>setFleet(e.target.value)}
```

Comment:
Can not be the same setFleet for min and max Fleet, because when you type in max Fleet input box, then the value is entered in min Fleet input box also.